package.json

```diff
@@ -10,6 +10,7 @@
     "test:components:all": "turbo test:components test:integration --concurrency 1",
     "format:all": "prettier --write .",
     "format:check:all": "prettier --check .",
+    "stylelint": "stylelint \"packages/sit-onyx/dist/*.css\"",
```

Comment:
@JoCa96 I am not sure if should mix up build and lint. What do you think about adding a separate styling step to the PR check pipeline?

Reply:
I meant that running `stylelint` depends on `build`, so this should be considered by turbo.
In the pipeline we can do it as part of the linting :)

Reply:
I added stylelint to check.yml

- should we still want to add it to turbo ?
- stylelint creates only warnings currently on standard output. That's also written to a text file. Do you have an idea what to do with the file ? 😃 Not sure about the possibilities here ..

Reply:
I would assume the command gives an appropriate exit code that we can use in the CI and local checks?

Reply:
yes!
I pushed a small update.
So the current state now is: 

- stylelint is run in the pipleline as seperate check and returns **errors** if it finds unsupported CSS features
- right now there are errors because of some features we use like _"css-selection"_
- we can get rid if the errors by 1) adding the feature to the **ignore array** in stylelint.config.js or by changing the **browserslist** or by non using the **feature** :-) 

we could also "mute" stylelint by turning errors into warnings but then why add it in the first place :-)